python-package-helpers.cmake

```diff
@@ -1,3 +1,80 @@
+function (ev_add_pip_package)
```

Comment:
If this function means "adding python package to cmake" we could call the `ev_create_pip_install_targets` function as a part of it, for python packages that are not installed during configuration